encoding/marshal.go

```diff
@@ -347,6 +348,19 @@ func Unmarshal(b []byte, v interface{}) error {
 	return NewDecoder(r).Decode(v)
 }
 
+// UnmarshalAll decodes the encoded values in b and stores them in vs, which
+// must be pointers.
+func UnmarshalAll(b []byte, vs ...interface{}) error {
```

Comment:
this function wound up not being used, but if we have a `MashalAll` it makes sense to have an `UnmarshalAll` for parity.